Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseTeleOp.java

```diff
@@ -0,0 +1,59 @@
+package org.firstinspires.ftc.team6220_PowerPlay;
```

Comment:
This file BaseTeleOp is the same one that Gavin already has a pull request out for.  You successfully tested it with him at the shop yesterday.  Please wait for his pull request to complete before merging this file.

Reply:
Yes, this pull request will only be merged after coordinating with Gavin